django/core/serializers/json.py

```diff
@@ -108,6 +109,8 @@ def default(self, o):
             return r
         elif isinstance(o, decimal.Decimal):
             return str(o)
+        elif isinstance(o, uuid.UUID):
+            return str(o)
```

Comment:
I don't mind what Claude proposed. I think it would be better to test this at a lower level, probably in `tests/serializers`. That should help verify there aren't any uuid issues with the other serializers too.